src/CommitmentsV2/SFA.DAS.CommitmentsV2.MessageHandlers/SFA.DAS.CommitmentsV2.MessageHandlers.csproj

```diff
@@ -1,10 +1,50 @@
-﻿<Project Sdk="Microsoft.NET.Sdk">
-
+﻿<Project Sdk="Microsoft.NET.Sdk;Microsoft.NET.Sdk.Publish">
   <PropertyGroup>
     <OutputType>Exe</OutputType>
     <LangVersion>latest</LangVersion>
     <TargetFramework>netcoreapp2.2</TargetFramework>
-    <TreatWarningsAsErrors>true</TreatWarningsAsErrors>
+    <TreatWarningsAsErrors>false</TreatWarningsAsErrors>
```

Comment:
Are we getting warnings that we can't resolve? Is that why this has been disabled?

Reply:
testing pulling in incompatible nuget packages